test/WebJobs.Script.Tests/ScriptHostTests.cs

```diff
@@ -1064,16 +1066,27 @@ public async Task Initialize_LogsWarningForExplicitlySetHostId()
             };
 
             var host = new HostBuilder()
+                .ConfigureAppConfiguration(b =>
+                {
+                    b.AddInMemoryCollection(new Dictionary<string, string>
+                    {
+                        { ConfigurationSectionNames.HostIdPath, "foobar" }
+                    });
+                })
                 .ConfigureDefaultTestWebScriptHost(_ => { },
                 options =>
                 {
                     options.ScriptPath = rootPath;
                     options.LogPath = Path.GetTempPath();
-                }, false,
+                }, true,
```

Comment:
What's this bool do?

Reply:
I'll add the explicit parameter name here, but this prevents automatic service initialization (removing the hosted services)